### ISSUE TITLE
fixes

### ISSUE THREAD

**Issue by junkyul** [junkyul/allenel#8]
trained weight or vocab.tar.gz
A field that importing vectors from a file or other sources
read pickle file and save as another text file (extract tar.gz or see load, save )
this might also help sparse processing?

model.tar.gz
see load_archive

Embedding in CPU/GPU [bring to GPU only relevant parts...]
Bag of word Embedding

SRL model prediction; do additional processings while predicting labels or in a demonstration

minor things about reading data set and vocab, cuda memory error logs

**Comment by junkyul** [comment]
types as MultiLabelField; returns one-hot encoded tensors
pass type embedder 
P(t | e) by sigmoid( vt * ve);
loss is 
log sum of 
P(t|e)'s for entity types (multiple types) and
(1 - P(t'|e) for other types
divide by the batch size

it's a mean log sum of sigmoid(v_i * v_e) and 1-sigmoid(v_i * v_e) ==> E 
use v_m in the place of v_e ==> T 

loss function sigmoid_cross_entropy_with_logits
https://discuss.pytorch.org/t/equivalent-of-tensorflows-sigmoid-cross-entropy-with-logits-in-pytorch/1985/11
https://stackoverflow.com/questions/34240703/what-is-logits-softmax-and-softmax-cross-entropy-with-logits/39499486#39499486
https://stackoverflow.com/questions/46218566/is-there-an-equivalent-loss-function-in-pytorch-to-tf-nn-softmax-cross-entropy-w

BCEWithLogitsLoss
